op.hcl

```diff
@@ -0,0 +1,26 @@
+// https://developer.1password.com/docs/cli/
+// https://app-updates.agilebits.com/product_history/CLI2
+description = "1Password CLI allows you to automate administrative tasks, securely provision secrets across development environments, and use biometrics to authenticate in the terminal."
+binaries = ["op"]
+test = "op --version"
+
+darwin {
+    source = "https://cache.agilebits.com/dist/1P/op2/pkg/v${version}/op_apple_universal_v${version}.pkg"
+
+    on "unpack" {
+        // How "${HOME}/usr/local/bin/op" was determined:
+        // 1. curl https://cache.agilebits.com/dist/1P/op2/pkg/v2.18.0/op_apple_universal_v2.18.0.pkg --output op_v2.18.0.pkg
+        // 2. pkgutil --expand op_v2.18.0.pkg op-v2.18.0/
+        // 3. grep install-location op-v2.18.0/op.pkg/PackageInfo
+        rename {
+            from = "${HOME}/usr/local/bin/op"
```

Comment:
🤔 
1. will hermit always use `-target CurrentUserHomeDirectory` for `installer` or only when testing? 
2. should it try to [delete](https://cashapp.github.io/hermit/packaging/schema/delete/) this `usr/local/bin` path?
3. if so, how is ordering controlled? put it after `rename`? in a different [trigger](https://cashapp.github.io/hermit/packaging/reference/#triggers-and-actions) like `install`?

Reply:
1. [Yes](https://github.com/cashapp/hermit/blob/ec480fe72b19d17150b49b92cf719332e86dad33/archive/archive.go#L350-L373) it will.
2. I'm not sure, this all sounds a bit sketchy really doesn't it?
3. Ordering is top-to-bottom.

Reply:
As nice as it would be to properly clean up, I'm not sure it can be done confidently (maybe someone wants an empty `$HOME/usr` directory for reasons) so I added a message for anyone looking to stay tidy.

I opened https://github.com/1Password/shell-plugins/issues/349 to try to get in touch with their developers about this CLI install location.